cpio/cpio.h

```diff
@@ -114,7 +114,7 @@ enum {
 	OPTION_QUIET,
 	OPTION_UUENCODE,
 	OPTION_VERSION,
-	OPTION_ZSTD,
+	OPTION_ZSTD
```

Comment:
I would leave the trailing commas, unless your compiler _really_ does not like them.